src/core/mavsdk_impl.cpp

```diff
@@ -133,6 +154,19 @@ bool MavsdkImpl::send_message(mavlink_message_t& message)
     return true;
 }
 
+void MavsdkImpl::process_heartbeat(const mavlink_message_t& message)
+{
+    std::lock_guard<std::mutex> lock(_connections_mutex);
+
+    uint16_t node_id = (message.sysid << 8) | message.compid;
+    std::cout << "HB Sysid: " << int(message.sysid) << " compid: " << int(message.compid) << std::endl;
```

Comment:
We don't `cout`, we log at some level. `LogErr() <<`,  `LogInfo() <<`, `LogDebug() <<`, ...

Reply:
Got it